romancal/datamodels/referencefile.py

```diff
@@ -0,0 +1,43 @@
+from stdatamodels import DataModel
+# Saved for validation
+# from .validate import ValidationWarning
+
+
+class RomanReferenceFileModel(DataModel):
+    """
+    Data model for reference files.
+    """
+    schema_url = "http://stsci.edu/schemas/roman_datamodel/reference_files/referencefile.schema"
+
+    def __init__(self, init=None, **kwargs):
+        super(RomanReferenceFileModel, self).__init__(init=init, **kwargs)
+        # self._no_asdf_extension = True
+        self.meta.telescope = "Roman"
+
+    # Saved for validation
+    # def validate(self):
+    #     """
+    #     Convenience function to be run when files are created.
+    #     Checks that required reference file keywords are set.
+    #     """
+    #     to_fix = []
+    #     to_check = ['telescope','description', 'reftype', 'author', 'pedigree', 'useafter']
```

Comment:
We have an [open issue in stdatamodels](https://github.com/spacetelescope/stdatamodels/issues/7) to fix validation of the `required` schema property.  After that work is done we should be able to use the schema for all this validation.

Reply:
Removed.